_content/lorawan/adaptive-data-rate.md

```diff
@@ -3,7 +3,7 @@ title: Adaptive Data Rate
 section: Specifications
 zindex: 7
 redirect_from:
- - current/adr/
+ - /lorawan/adr
```

Comment:
Did you test that this creates a redirect from `/lorawan/adr.html`? Otherwise it doesn't work